File: 0-Warmup/Staircase/code.c
/*
 * Complete the 'staircase' function below.
 *
 * The function accepts INTEGER n as parameter.
 */

void staircase(int n) {
    for(int i=0; i<n; ++i){
        for(int j=0; j<n-1-i; ++j) printf(" ");
        for(int z=0; z<i+1; ++z) printf("#");
        printf("\n");
    }
}

File: 1-Implementation/Breaking the Records/code.py
#
# Complete the 'breakingRecords' function below.
#
# The function is expected to return an INTEGER_ARRAY.
# The function accepts INTEGER_ARRAY scores as parameter.
#

def breakingRecords(scores):
    Max = scores[0]
    Min = scores[0]
    maxCount = 0
    minxCount = 0

    for i in scores:
        if i > Max:
            Max = i
            maxCount = maxCount + 1
        elif i < Min:
            Min = i
            minxCount = minxCount + 1

    return maxCount, minxCount
